### PR TITLE
[Tron]: Signing message 

### PR DESCRIPTION
**Pull request by Milerius**
## Description

Needed for tron dapp support

## How to test

Run C++ unit tests

### CODE REVIEW

File: src/Aptos/MoveTypes.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright © 2017-2022 Trust Wallet.
+// Copyright © 2017-2023 Trust Wallet.
```
Comment (rsrbk) [inline src/Aptos/MoveTypes.cpp:1]: Maybe next time it's better to submit this kind of changes as a separate PR, cause it's really hard to find what to review here

**Review by rsrbk — APPROVED** [review]

**Comment by rsrbk** [comment]
Just curious, is there a big difference in the way to sign message from Tron comparing to other EVMs?

**Comment by Milerius** [comment]
> Just curious, is there a big difference in the way to sign message for Tron comparing to other EVMs?

Not so big, message length is hardcoded (32). - The prefix msg is different TRON instead of ETH.

File: src/Aptos/MoveTypes.cpp (hunk 1 above)
Comment (Milerius) [reply src/Aptos/MoveTypes.cpp:1]: You can review commit by commit in the review

Comment (Milerius) [reply src/Aptos/MoveTypes.cpp:1]: Screenshot 2023-03-14 at 14 45 33

**Review by satoshiotomakan — APPROVED** [review]

**Review by hewigovens — APPROVED** [review]
only change copyright year when you edit that file, too many unrelated changes

**Review by vcoolish — APPROVED** [review]

**Comment by D777Del** [comment]
D777del
